client/src/components/PinnedFAQ/PinnedFAQ.tsx

```diff
@@ -18,7 +18,20 @@ import { secondaryColor } from "src/style_constants/colors.interop.scss";
 export const SOME_THINGS_TO_KEEP_IN_MIND_STORAGE_KEY =
   "user_enabled_pinning_key_concepts";
 
-export class PinnedFAQ extends React.Component {
+const PinnedFAQDefaultProps = {
+  background_color: secondaryColor,
+  is_initially_expanded: false,
+  subject: Gov.instance,
+};
+
+type PinnedFAQProps = typeof PinnedFAQDefaultProps & {
+  q_a_key_pairs: Array<Array<string>>;
```

Comment:
In this case, I would just recommend to use `string[][]` to keep our coding style consistent

Reply:
Sounds good, I updated the file and made the change. I'll also keep it in mind for the future!